server/src/main/java/io/horizen/Main.java

```diff
@@ -0,0 +1,82 @@
+package io.horizen;
+
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+import io.horizen.config.AppModule;
+import io.horizen.config.Settings;
+import io.horizen.helpers.Helper;
+import io.horizen.services.SnapshotService;
+import io.horizen.utils.Balancer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.net.ssl.KeyManagerFactory;
+import javax.net.ssl.SSLContext;
+import java.io.FileInputStream;
+import java.security.KeyStore;
+
+import static spark.Spark.*;
+
+
+public class Main {
+
+    private static final Logger log =  LoggerFactory.getLogger(Main.class);
+
+    public static void main(String[] args) {
+        Settings settings = new Settings();
+
+        setupSSL(); //turn off for local dev
+
+        port(5000); //change for different port
+
+        checkMocks(settings);
+
+        Helper.initialize(settings);
+
+        Injector injector = Guice.createInjector(new AppModule(settings));
+        Balancer balancer = injector.getInstance(Balancer.class);
+        balancer.setupRoutes();
+
+        SnapshotService snapshotService = injector.getInstance(SnapshotService.class);
+        snapshotService.initActiveProposal();
+
+        // Stop the server gracefully when the application shuts down
+        Runtime.getRuntime().addShutdownHook(new Thread(() -> {
+            stop();
+            System.out.println("Server stopped");
+        }));
+    }
+
+    private static void setupSSL() {
+        String keystoreFilePath = "/home/ddrvar/keystore/keystore.jks";
```

Comment:
I guess this is temporary. Probably in the final version we could add a definition in the settings should https ever be needed

Reply:
Done, same env vars as in [this PR](https://github.com/HorizenLabs/balancer/pull/3)